test/alternator/test_table.py

```diff
@@ -452,7 +452,7 @@ def check_pre_consistent_cluster_management(dynamodb):
     from util import is_aws
     # If not running on Scylla, return false.
     if is_aws(dynamodb):
-        return false
```

Comment:
Is this change connected with this PR?

Reply:
No, seems occasional change. I'll revert this.

Reply:
Oops, thanks I'll send a separate patch to fix that. I need to check how I could have missed this - I guess I didn't run any of the tests that need this obscure fixture on AWS DynamoDB.

Reply:
https://github.com/scylladb/scylladb/pull/17799